### ISSUE TITLE
שאלה כללית לגבי יצירת רשימות וי.או.די + תמונות

### ISSUE THREAD

**Issue by burekas7** [Fishenzon/repo#93]
אני אישית משתמש בסקין טיטן.
אני משתמש בווידג'ט שנמצא במסך הראשי, וכאשר נמצאים על אייטם מסוים של אותו וידג'ט אז הרקע משתנה בהתאם אליו.
התוכן של הווידג'ט מוגדר להציג את האייטמים שאני שומר בקטגוריה של "מועדפי עידן פלוס" שבתוך ההרחבה.

עכשיו יש לי בעיה מסוימת עם אייטמים של תוכניות שהם נשמרו מהוי.או.די של רשת בתוך "מועדפי עידן פלוס".
כאשר אני עומד ועובר בין אייטמים של תוכניות מ"קשת" או "כאן" אז הרקע משתנה בהתאם,
אבל כשאני עובר לאייטם שהוא תוכנית של "רשת" אז הרקע לא משתנה.

השאלה שלי האם התמונות שנלקחות עבור התוכניות של רשת נלקחות ומוגדרות בצורה שונה לאייטמים שלו מאשר התוכניות של קשת ו-כאן?
האם בכלל מוצמדת תמונת רקע לתוכניות רשת או שזה בעצם מוגדר כתמונות ממוזערות ולא כרקע.
אני לא יודע אם בהרחבה זה מוגדר כ-
fanart
של אותו אייטם, אבל זו הכוונה, לתמונת הרקע.

פשוט מנסה להבין איפה הבעיה.

תודה מראש

**Comment by Fishenzon** [comment]
אשתדל לבדוק אם משהו שונה בקובץ של רשת

**Comment by Fishenzon** [comment]
בוקר טוב.
לא ראיתי שוני בין התכנים של רשת לתכנים ממקורות אחרים.
אצלי זה מציג רקע בכל המקרים

**Comment by burekas7** [comment]
נכון, בתוך ההרחבה (נניח במועדפים) הרקע משתנה במעבר בין התוכניות השונות מהערוצים השונים (כולל רשת).
הבעיה היא כאשר בסקין שלי יש ווידג'ט על מסך הבית שמציג את התוכן של המועדפים, אז בתוכניות שמשויכות ל"רשת",
אז דווקא בהן הרקע לא משתנה משום מה.
לכן חשבתי אולי צורת שמירת התמונה והשיוך שלה לאייטם של התוכנית מתבצע אולי בצורה שונה, אולי נשמר כתמונה ממוזערת, אולי כרקע, אולי רזולוציה שונה משאר הערוצים וכו'.

צירפתי סרטון להמחשה שתבין את הכוונה שלי:
https://f2h.io/5myuebhlouet

אני מנסה לחקור את העניין גם מצד הסקין, אבל זה יותר מורכב.
יש מצבים מאוד מסוימים שהוא כן מציג את הרקע, אבל לא כמעבר ביניהם כפי שהצגתי בסרטון.
